src/main.cpp

```diff
@@ -15,16 +15,27 @@ int main(int argc, char** argv) {
         RIP("XDG_RUNTIME_DIR not set!");
 
     // parse some args
-    for (int i = 1; i < argc; ++i) {
-        if (!strcmp(argv[i], "--i-am-really-stupid"))
-            ignoreSudo = true;
-    }
+    if (!strcmp(argv[1], "i-am-really-stupid"))
+        ignoreSudo = true;
+    else if (!strcmp(argv[1], "help"))
```

Comment:
I think it would be easier to convert `argv` to `string` and use the associated `==` to compare them. This is C++ after all.

Reply:
I missed something about what I'm talking about